core/src/coproduct.rs

```diff
@@ -144,28 +144,361 @@ macro_rules! Coprod {
     // Forward trailing comma variants -->
 }
 
-/// Trait for instantiating a coproduct
+// Inherent methods
+impl<Head, Tail> Coproduct<Head, Tail> {
+    /// Instantiate a coproduct from an element.
+    ///
+    /// This is generally much nicer than nested usage of `Coproduct::{Inl,Inr}`.
```

Comment:
Space between `Inl,` and `Inr` ;)

Reply:
I'll do this, but until now I've always *deliberately* written it without the space since the `{a,b}` syntax comes from shell syntax, where a space would actually mess things up. Might be a tough habit to unlearn... :stuck_out_tongue: